model/task_queue.go

```diff
@@ -80,11 +79,10 @@ func (q *DistroQueueInfo) GetQueueCollection() string {
 
 // TaskQueue represents the next n tasks to be run on hosts of the distro
 type TaskQueue struct {
-	Id              mgobson.ObjectId `bson:"_id,omitempty" json:"_id"`
```

Comment:
Is it safe to remove this? I don't know what effect this will have on the database model (since `_id` is the unique identifier for the document and there are already existing queues in the db with `_id` defined).

Reply:
It isn't actually used very often, and I went through and modified all instances where it was.